src/plugins/tool_extrude.py

```diff
@@ -14,6 +14,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from PyQt4 import QtGui
```

Comment:
At least the whole project is written with PySide so far. No objections for switching over to PyQt, but pls not in this PR ;)